lib/sdf-server/src/service/module/upgrade_modules.rs

```diff
@@ -33,16 +39,52 @@ pub async fn upgrade_modules(
     OriginalUri(original_uri): OriginalUri,
     Host(host_name): Host,
     Json(request): Json<UpgradeModulesRequest>,
-) -> Result<ForceChangeSetResponse<Vec<FrontendVariant>>, ModuleError> {
+) -> ModuleResult<ForceChangeSetResponse<Ulid>> {
     let mut ctx = builder.build(request_ctx.build(request.visibility)).await?;
 
     let force_change_set_id = ChangeSet::force_new(&mut ctx).await?;
 
-    let mut variants = Vec::new();
+    let task_id = Ulid::new();
+
+    tokio::task::spawn(async move {
```

Comment:
I can't remember if we have a tracker in the app state but this should be fine for now
cc @fnichol